coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -1,3 +1,10 @@
+"""
+Language and docstyle independent documentation comments extraction.
```

Comment:
this still doesn't sound right, how about

`.... independent extraction of documenation comments.`